src/Agent.Plugins/Artifact/ArtifactItemFilters.cs

```diff
@@ -196,6 +196,26 @@ public List<FileContainerItem> ApplyPatternsMapToContainerItems(List<FileContain
             return resultItems;
         }
 
+        // Returns list of FileInfo items required to be downloaded. Used by FileShareProvider.
+        public List<FileInfo> ApplyPatternsMapToFileShareItems(IEnumerable<FileInfo> files, Hashtable map, string sourcePath)
```

Comment:
Could you please add comments in ///<summary> format?

Reply:
Done for all PRs about DownloadBuildArtifact with adding new methods